content/posts/ossprey-announcement/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Introducing OSSPREY: AI-Powered Forecasting and Actionable Insights for OSS Sustainability"
+date: 2025-09-11T10:00:00-07:00
+draft: false
+summary: "OSSPREY delivers AI-powered sustainability forecasting and evidence-based recommendations to help maintainers keep projects healthy."
+tags: ["sustainability", "open source", "forecasting", "AI"]
+authors: ["Nafiz Imtiaz Khan", "Priyal Soni", "Arjun Ashok", "Vladimir Filkov"]
+featuredImage: "https://raw.githubusercontent.com/OSS-PREY/OSSPREY-Website/refs/heads/main/static/images/ossprey-dashboard.png"
+---
+
+Open Source Software (OSS) powers everything from academic research to mission-critical infrastructure at tech giants. Yet, over **90% of OSS projects experience stagnation or abandonment**. The question is no longer whether OSS matters, but _how to ensure its sustainability_.
+
+## Meet OSSPREY
+
+The OSSPREY paper has been accepted in the [Automated Software Engineering 2025 Tool Demonstration Track](https://conf.researchr.org/track/ase-2025/ase-2025-tool-demonstration-track).
+
+![OSSPREY Icon](https://oss-prey.github.io/OSSPREY-Website/static/images/favicon.ico)
+
+**OSSPREY (Open Source Software PRoject sustainabilitY tracker)** is an intelligent, modular, and research-backed platform that enables open source maintainers to anticipate and address project health issues before they become critical. Developed by the DECAL Lab at UC Davis, OSSPREY uniquely combines real-time GitHub data scraping, socio-technical network modeling, machine learning forecasting, and a curated knowledge base of interventions grounded in software engineering research.
```

Comment:
I think the intro to the project should be the first line, and should include a link to the project, then mention the paper acceptance.